cmd/lambda/webhook/handler.go

```diff
@@ -50,3 +207,43 @@ func (h *Lambda) Handler(ctx context.Context, req events.APIGatewayProxyRequest)
 	adapter := handlerfunc.New(h.Server.ServeHTTP)
 	return adapter.ProxyWithContext(ctx, req)
 }
+
+type RecordingEventBody struct {
+	Data map[string]string `json:"data"`
+}
+
+func buildLocalHandler() (*Server, error) {
+	ctx := context.Background()
+	var cfg Config
+	err := envconfig.Process(ctx, &cfg)
+	if err != nil {
+		return nil, err
+	}
+	log, err := logger.Build(cfg.LogLevel)
+	if err != nil {
+		return nil, err
+	}
+	zap.ReplaceGlobals(log.Desugar())
+
+	s, err := NewServer(ctx, cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	return s, nil
+}
+
+func (s *Server) Run(ctx context.Context) error {
+
+	serv := &http.Server{
+		Addr:    "localhost:3030",
```

Comment:
As per my comment above in the Lambda file this should be shifted out - we shouldn't see any `localhost` references here as this file is for AWS Lambda